builders/deploy.yaml

```diff
@@ -18,22 +18,195 @@
 
 substitutions:
   _REGION:
-  _SERVICE:
+  _REPO: 'github.com/google/exposure-notifications-server'
   _TAG:
 
 steps:
-- id: 'deploy'
-  name: 'gcr.io/google.com/cloudsdktool/cloud-sdk:303.0.0-alpine'
+#
+# cleanup-exposure
+#
+- id: 'deploy-cleanup-exposure'
+  name: 'gcr.io/google.com/cloudsdktool/cloud-sdk:307.0.0-alpine'
+  args:
+  - 'bash'
+  - '-eEuo'
+  - 'pipefail'
+  - '-c'
+  - |-
+    gcloud run deploy "cleanup-exposure" \
+      --quiet \
+      --project "${PROJECT_ID}" \
+      --platform "managed" \
+      --region "${_REGION}" \
+      --image "gcr.io/${PROJECT_ID}/${_REPO}/cleanup-exposure:${_TAG}" \
+      --no-traffic
+  waitFor:
+  - '-'
+
+#
+# cleanup-exposure
+#
+- id: 'deploy-cleanup-exposure'
+  name: 'gcr.io/google.com/cloudsdktool/cloud-sdk:307.0.0-alpine'
+  args:
+  - 'bash'
+  - '-eEuo'
+  - 'pipefail'
+  - '-c'
+  - |-
+    gcloud run deploy "cleanup-exposure" \
+      --quiet \
+      --project "${PROJECT_ID}" \
+      --platform "managed" \
+      --region "${_REGION}" \
+      --image "gcr.io/${PROJECT_ID}/${_REPO}/cleanup-exposure:${_TAG}" \
+      --no-traffic
+  waitFor:
+  - '-'
+
+#
```

Comment:
Maybe we don't want to deploy this all the time? We don't have an easy way to do conditionals though...